src/templates/cv_medicine_template.html

```diff
@@ -601,27 +606,25 @@
     <% if (teaching_activities) { %>
     <div class="row">
     <div class="heading_10">EDUCATION / TEACHING ACTIVITIES <br>
-      <span class="regular_10_italic">Describe activities teaching and providing educational support for residents, medical students, and continuing medical education. List activities by:
-          <ul>
-          <li><p>Teaching and supporting learners</p></li>
-          <li><p>Development of courses / educational programs</p></li>
-          <li><p>Development of assessment tools and methods</p></li>
-          <li><p>Education management / leadership</p></li>
-        </ul></span></div>
+      <span class="regular_10_italic">Describe activities teaching and providing educational support for residents, medical students, and continuing medical education. List activities by:<br/>
+          &bull;&nbsp;&nbsp;Teaching and supporting learners<br/>
```

Comment:
This is sad - having to put in `&bull;&nbsp;` instead of just using a `<ul><li>`.  I would guess you tried other stuff though.